app/assets/stylesheets/activeadmin_addons/addons/material-toggle_bool.scss

```diff
@@ -0,0 +1,53 @@
+$inactive-button: #aaa;
+$inactive-bg: #ddd;
+$active-button: $am-theme-primary;
+$active-bg: rgba($active-button, .6);
+
+$box-shadow: 0 2px 1px -1px rgba(0,0,0,.2), 0 1px 1px 0 rgba(0,0,0,.14), 0 1px 3px 0 rgba(0,0,0,.12);
+
+.toggle-bool-switches-container {
+  text-align: center;
+
+  .toggle-bool-switch {
+    cursor: pointer;
+
+    background: $inactive-bg;
+    background-image: none;
+    box-shadow: $box-shadow;
+
+    width: 36px;
+    height: 14px;
+    display: inline-block;
+    position: relative;
+    border-radius: 9px;
+
+    
```

Comment:
Unexpected whitespace at end of line                           no-eol-whitespace